token/cli/src/main.rs

```diff
@@ -3624,6 +3652,48 @@ fn app<'a, 'b>(
                 .nonce_args(true)
                 .offline_args(),
         )
+        .subcommand(
+            SubCommand::with_name(CommandName::UpdateMetadataAddress.into())
+                .about("Updates metadata pointer address for the mint. Requires the metadata pointer extension.")
+                .arg(
+                    Arg::with_name("token")
+                        .validator(is_valid_pubkey)
+                        .value_name("TOKEN_MINT_ADDRESS")
+                        .takes_value(true)
+                        .index(1)
+                        .required(true)
+                        .help("The address of the token mint to update the metadata pointer address"),
+                )
+                .arg(
+                    Arg::with_name("metadata_address")
+                        .index(2)
```

Comment:
Can you add `.validator(is_valid_pubkey)` for this arg too? Otherwise, if someone inputs an invalid pubkey, it'll disable the metadata pointer address!

Reply:
Yupp, Done! and also sorry about this many commits. :)